### PR TITLE
ADD: Conditionally turn off lm

### PR DESCRIPTION
**Pull request by benleetownsend**

### CODE REVIEW

**Comment by benleetownsend** [comment]
UNTESTED: Suspect issues with saving and loading.

**Comment by benleetownsend** [comment]
all tests passed.

File: finetune/base.py [hunk 1]
```diff
@@ -438,26 +438,29 @@ def _construct_graph(self, n_updates_total, target_dim=None, train=True, pre_tra
...
-                    hidden=featurizer_state['sequence_features'],
-                    reuse=do_reuse
-                )
-
                 lm_loss_coef = self.config.lm_loss_coef
                 if target_dim is None:
                     lm_loss_coef = 1.0
 
-                train_loss = lm_loss_coef * tf.reduce_mean(language_model_state['losses'])
+                if (train and lm_loss_coef > 0) or self.require_lm:
```
Comment (madisonmay) [inline finetune/base.py]: make into single boolean that's used twice

**Comment by madisonmay** [comment]
Reopening into dev

**Pull request closed without merging**